sdk/src/Services/MobileAnalytics/Custom/_unity/BackgroundRunner.unity.cs

```diff
@@ -104,6 +104,10 @@ private void DoWork()
                     }
                     Thread.Sleep(BackgroundSubmissionWaitTime * 1000);
                 }
+                catch(ThreadAbortException)
+                {
+                  // handle thread aports more greacefully in unity
```

Comment:
@aphexyuri Please use _logger.Debug() instead of _logger.Error(), this increase the logger level to remind developers should fix the issue.